packages/ui/src/components/Visualization/Canvas/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './Canvas';
 export * from './CanvasSideBar';
-export * from './DataFormatEditor';
-export * from './StepExpressionEditor';
+export * from '../../Form/dataFormat/DataFormatEditor';
+export * from '../../Form/stepExpression/StepExpressionEditor';
```

Comment:
mmm, this is interesting, I think we should remove these exports from this file and export them from the `Form` folder instead, let me check

Reply:
Seems this is not used, so we can delete these exports.

Reply:
I tried removing them but found out that they are getting used [here](https://github.com/KaotoIO/kaoto-next/tree/main/packages/ui-tests/stories/metadataEditor). So I moved those exports to the `Form` folder. @lordrip

Reply:
Good catch 👌, that helps